src/welldev/io/model/ProductList.java

```diff
@@ -0,0 +1,37 @@
+package welldev.io.model;
+
+public class ProductList {
+	
+	int id;
+
+	String name;
+	int quantity;
+	float price;
```

Comment:
you can also add private modifiers it looks nice :)

Reply:
@kud-wtag it is not for looking nice, it is necessary :)